dask_expr/collection.py

```diff
@@ -96,6 +96,10 @@ def _meta(self):
     def size(self):
         return new_collection(self.expr.size)
 
+    @property
+    def columns(self):
```

Comment:
My preference is to stick with exactly what you have here, but just a thought: Do we want to raise an error for `Series.columns` (to be consistent with pandas)?

Reply:
>We can add this here, this wouldn't break any tests

Up to you. This PR is already a win "as is" in my mind.

Reply:
Then let's merge as is, the columns change isn't urgent